src/main/kotlin/com/learnspigot/bot/verification/VerificationListener.kt

```diff
@@ -162,33 +156,33 @@ class VerificationListener : ListenerAdapter() {
 
                 "u" -> {
                     val originalActionTaker = info[4]
-                    if (e.member!!.id != originalActionTaker && !roleIds.contains(Environment.get("MANAGEMENT_ROLE_ID"))) {
-                        e.reply("Sorry, you can't undo that verification decision.").setEphemeral(true).queue()
-                        return
+
+                    if (clicker.id != originalActionTaker && !clicker.isManager) {
+                        return e.reply("Sorry, you can't undo that verification decision.").setEphemeral(true).queue()
```

Comment:
Minor thing, but I think you'd want to use replyEphemeral here instead of reply("...").setEphemeral(true).queue()

Reply:
I would use #reply_, see: 

```kt
    event.reply_(
        "Are you sure you want to ban **${user.asTag}**?",
        components=confirm.into(),
        ephemeral=true
    ).queue()
```